rotkehlchen/types.py

```diff
@@ -1081,6 +1082,10 @@ class CacheType(Enum):
     COINLIST = auto()  # coinlist / all coins cache for various oracles
     AIRDROPS_METADATA = auto()  # airdrops index fetched from rotki/data repo
     AIRDROPS_HASH = auto()  # hash of airdrops csv file
+    GEARBOX_POOL_ADDRESS = auto()
```

Comment:
Perhaps not for this PR but can't we generalize the types somehow? Since I see many caches for pool address, with main difference beign the protocol. Or think this makes no sense in the context we use this cache?